homeassistant/components/homekit/type_covers.py

```diff
@@ -135,6 +174,21 @@ def update_state(self, new_state):
         else:
             self.char_position_state.set_value(2)
 
+        # update tilt
+        current_tilt = new_state.attributes.get(ATTR_CURRENT_TILT_POSITION)
+        if isinstance(current_tilt, (float, int)):
+            # HomeKit sends values between -90 and 90.
+            # We'll have to normalize to [0,100]
+            current_tilt = (current_tilt / 100.0 * 180.0) - 90.0
+            current_tilt = int(current_tilt)
+            self.char_current_tilt.set_value(current_tilt)
+
+            if self._homekit_target_tilt is None or abs(
+                current_tilt - self._homekit_target_tilt < 1
+            ):
+                self.char_target_tilt.set_value(current_tilt)
+                self._homekit_target = None
```

Comment:
```suggestion
                self._homekit_target_tilt = None
```

I'm not sure this is right.